Makefile

```diff
@@ -40,7 +40,7 @@ lint-apply: lint-black-apply lint-isort-apply ## apply all the linter's suggesti
 
 .PHONY: test
 test:  ## run tests in parallel
```

Comment:
Let's remove the "run test in parallel" comment.

Reply:
Sure, if you prefer it. I have changed it in [495d736](https://github.com/freedomofpress/dangerzone/pull/288/commits/495d73671ab7ea36b4b7f70a1050aa2d1f5f0169).

`run_tests_parallel()` would also need to follow the same logic so I changed it to `run_tests_in_parallel` and `run_tests_in_sequence()`, respectively. Does that sound good?